src/NuGet.Services.EndToEnd/Support/Clients/RegistrationClient.cs

```diff
@@ -238,6 +292,20 @@ public class CatalogEntry
             public string LicenseExpression { get; set; }
             public string LicenseUrl { get; set; }
             public string IconUrl { get; set; }
+            public CatalogDeprecation Deprecation { get; set; }
+        }
+
+        public class CatalogDeprecation
+        {
+            public string Message { get; set; }
+            public IEnumerable<string> Reasons { get; set; }
```

Comment:
nit: List<string>/IReadOnlyList<string> for easier indexing and clear enumeration

Reply:
done